### PR TITLE
Lint all the things [ci: last-only]

### PR DESCRIPTION
**Pull request by som-snytt**

### CODE REVIEW

**Comment by som-snytt** [comment]
Added `internal` label even though it results in user-visible increase in quality.

**Comment by som-snytt** [comment]
Needs a tweak for commented code, and also a check that subprojects are correctly linted (!). But mostly, apparently, this is just to bring scalacheck tests mostly into lint conformance.

File: test/files/pos/t7591/Demo.scala [hunk 1]
```diff
@@ -38,10 +38,10 @@ trait DemoSpec extends Spec with Meta.StdOpts with Interpolation {
 
   heading("Binary options:")
   val optopt          = "str"       / "an optional String"        --|
-  val optoptInt       = ("int"      / "an optional Int") .        --^[Int]
+  //val optoptInt       = ("int"      / "an optional Int") .        --^[Int]
```
Comment (lrytz) [inline test/files/pos/t7591/Demo.scala]: these changes should be in the other PR

**Review by lrytz — APPROVED** [review]
All good otherwise, thanks!

**Comment by som-snytt** [comment]
De-commented the test.

Scalatest doesn't need `-Wmacros:after` for `Liftable` after the timely change to support `-Wmacros:default` which sees the usage of the liftable in the expansion. The bootstrapped run should have no warning with `-Werror`; non-bootstrapped should see a warning but without `-Werror`.

**Pull request merged**